example.py

```diff
@@ -6,15 +6,14 @@
 
 from haikunator import Haikunator
 
```

Comment:
This should fix the error I mentioned in the main conversation. This renames `raw_input()` to `input()` for Python 2 (necessary for line 238 to work).
```suggestion
from six.moves import input

```